organization/components/timeslot/TimeslotForm.tsx

```
@@ -0,0 +1,163 @@
+            className="w-full px-3 py-2 border border-gray-300 rounded-lg focus:ring-2 focus:ring-blue-500 focus:border-blue-500 outline-none transition"
+            min={new Date().toISOString().split('T')[0]}
+            required
+          />
+        </div>
+
+        {/* End Date */}
+        <div className="space-y-1">
+          <label className="text-sm font-medium text-gray-700 flex items-center gap-2">
+            <FiCalendar className="text-blue-600" />
+            End Date
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)